src/services/database.ts

```diff
@@ -93,3 +105,51 @@ const CriteriaIndexMap: Record<Exclude<SearchCriteria, SearchCriteria.ALL>, Tabl
   vin: 'VinIndex',
   trailerId: 'TrailerIdIndex',
 };
+export const postTechRecord = async (request: any) => {
+  const systemNumber = await generateNewNumber(NumberTypes.SystemNumber);
```

Comment:
I would like to extract this logic to the handler level. Ideally I would like the database methods to just do database work. You pass it a record, it saves it, end of work.

Reply:
I think also a helper function from the handler would help, to lower code complexity inside that